lib/bedrock.js

```diff
@@ -107,6 +107,8 @@ api.start = function(options, done) {
       'Override console log timestamps config. (boolean)', brUtil.boolify)
     .option('--log-colorize <colorize>',
       'Override console log colorization config. (boolean)', brUtil.boolify)
+    .option('--log-exclude <modules>',
+      'Do not log events from the specified comma seperated modules.')
```

Comment:
typo: `seperated` => `separated` ... this is a problem for the description in `log-only` as well.